cli/cli/helpers/grafloki/docker_grafloki.go

```diff
@@ -146,6 +148,7 @@ func createGrafanaAndLokiContainers(ctx context.Context, dockerManager *docker_m
 		WithRestartPolicy(docker_manager.RestartOnFailure).
 		WithNetworkMode(bridgeNetworkName).
 		WithLabels(grafanaContainerLabels).
+		WithUser(root).
```

Comment:
This is definitely a permission escalation which - while common in the Docker ecosystem - is best avoided if possible. It's 98% probably fine, but if you want to do it the secure way you could consider:
- Using Bash file embedding (treats strings as if they were file, a la `<(echo "foobar")`).
- Configure using envvars, if possible